packages/template-retail-react-app/app/components/toggle-card/index.jsx

```diff
@@ -34,6 +34,12 @@ export const ToggleCard = ({
     children,
     ...props
 }) => {
+    let editButtonAriaLabel = 'Edit'
+    if (typeof editLabel === 'string') {
+        editButtonAriaLabel = editLabel
+    } else if (typeof editLabel === 'object' && editLabel?.type?.displayName === 'MemoizedFormattedMessage') {
+        editButtonAriaLabel = editLabel?.props.defaultMessage[0].value
+    }
```

Comment:
## Need advice on this. This feels hacky.

A better solution would be to restrict editLabel from `any` to `string` so we can pass it directly to `aria-label` on the button and handle translation for `editLabel` on the button inside `ToggleCard`.

But changing type from `any` to `string` will be a breaking change.